apps/namehashlabs.org/data/ensProfiles.ts

```diff
@@ -467,7 +467,7 @@ const EnsProfiles = new Map<`0x${string}`, Profile>([
   [
     "0x89F8e4020c0dd384F13c288bc5743F963F9D8fdF",
     buildProfile({
-      ensName: "edulennert.eth",
+      ensName: "duds.eth",
```

Comment:
@eduramme Could you please set the avatar on your ENS profile?

![CleanShot 2024-11-09 at 21 09 21](https://github.com/user-attachments/assets/4bcc41ca-a7f7-4a1f-a551-4b55de31ec81)

I see you never set any avatar:

![CleanShot 2024-11-09 at 21 10 20](https://github.com/user-attachments/assets/8986ac93-7d8a-4097-9b5d-ee71911864c4)